docs/assets/templates/.dockstore.yml

```
@@ -7,13 +7,14 @@ service:
   subclass:
   name:
 
-  # Optional array of authorship information, requires at least the name of each author.
+  # Optional array of authorship information, requires at least the name or orcid of each author.
+  # If an author's ORCID iD is provided, other authorship fields should not be included because they will be retrieved from their ORCID record.
```

```suggestion
  # If an author's ORCID ID is provided, other authorship fields will be ignored, as that information will be retrieved from their ORCID record.
```

Slight clarification that the information is simply ignored